velox/docs/functions/spark/datetime.rst

```diff
@@ -187,6 +187,13 @@ These functions support TIMESTAMP and DATE input types.
     Returns the `ISO-Week`_ of the year from x. The value ranges from ``1`` to ``53``.
     A week is considered to start on a Monday and week 1 is the first week with >3 days.
 
+.. function:: weekday(date) -> integer
+
+    Returns the day of the week for date (0 = Monday, 1 = Tuesday, …, 6 = Sunday).
```

Comment:
Having both weekday and dayofweek functions that sound similar but return different results is confusing. Do you ever hear complains from users?

Reply:
It is confusing, I suppose different users use different functions depending on the indexing they're looking for.